source/rst/continuous_time/seir_model_sde.rst

```diff
@@ -0,0 +1,913 @@
+
+This is a common trick when solving systems of ODEs.  While equivalent in principle to using the appropriate quadrature scheme, this becomes especially important and convenient when adaptive time-stepping algorithms are used to solve the ODEs (i.e. there is no fixed time grid).
+
+The system :eq:`seir_system` and the supplemental equations can be written in vector form :math:`x := [s, e, i, r, R, m, c, M]` with parameter tuple :math:`p := (\sigma, \gamma, B, \eta)`
+
+.. math::
+    \begin{aligned} 
+    \frac{d x}{d t} &= F(x,t;p)\\
+        &:= \begin{bmatrix}
+        - \gamma \, R \, s \,  i  
+        \\
+        \gamma \, R \,  s \,  i  - \sigma e 
+        \\
+        \sigma \, e  - \gamma i
+        \\
+        \gamma i
+        \\
+         \eta (B(t) - R)
+        \\
+        0        
+        \\
+        \sigma e
+        \\
+        m \, \gamma \, i
+        \end{bmatrix}
+    \end{aligned}         
+    :label: dfcv
+
+Here note that if :math:`B(t)` is time-invariant, then :math:`F(x)` is time-invariant as well. 
+
+Parameters
+----------
+
+Both :math:`\sigma` and :math:`\gamma` are thought of as fixed, biologically determined parameters.
+
+As in Atkeson's note, we set
+
+* :math:`\sigma = 1/5.2` to reflect an average incubation period of 5.2 days.
+* :math:`\gamma = 1/18` to match an average illness duration of 18 days.
+* :math:`B = R = 1.6` to match an **effective reproduction rate** of 1.6, and initially time-invariant
+* :math:`m(t) = m_0 = 0.01` for a one-percent mortality rate
+
+
+As we will initially consider the case where :math:`R(0) = B`, the value of :math:`\eta` will drop out of this first experiment.
+
+Implementation
+==============
+
+
+First, construct our :math:`F` from :eq:`dfcv`
+
+.. code-block:: julia
+
+    function F(x, p, t)
+
+        s, e, i, r, R, m, c, M = x
+        @unpack σ, γ, B, η = p
+
+        return [-γ*R*s*i;       # ds/dt
+                γ*R*s*i -  σ*e; # de/dt
+                σ*e - γ*i;      # di/dt
+                γ*i;            # dr/dt
+                η*(B(t, p) - R);# dR/dt
+                0.0;            # dm/dt
+                σ*e;            # dc/dt
+                m*γ*i;          # dM/dt
+                ]          
+    end
+
+
+Parameters
+-------------
+
+The baseline parameters are put into a named tuple generator (see previous lectures using ``Parameters.jl``) with default values discussed above.  
+
+.. code-block:: julia
+
+    p_gen = @with_kw ( T = 550.0, γ = 1.0 / 18, σ = 1 / 5.2, η = 1.0 / 20,
+                      R̄ = 1.6, B = (t, p) -> p.R̄, m_0 = 0.01, N = 3.3E8)
+
+Note that the default :math:`B(t)` function always equals :math:`\bar{R}`
+
+
+Setting initial conditions, we will assume a fixed :math:`i, e`, :math:`r=0`, :math:`R(0) = \bar{B}`, and :math:`v(0) = 0.01` 
```

Comment:
Not sure what `v` is supposed to be referring to.

Reply:
Sorry, not on computer, but it might be old notation for the death probability in the python ? Maybe I changed it to `m(t) ` for mortality?